packages/cfd/src/Components/cfd-personal-details-form.tsx

```diff
@@ -439,11 +430,9 @@ const CFDPersonalDetailsForm = ({
                                 <Modal.Footer is_bypassed={isMobile()}>
                                     {form_error && <FormSubmitErrorMessage message={form_error} />}
                                     <FormSubmitButton
-                                        cancel_label={localize('Previous')}
                                         is_disabled={isSubmitting || !isValid}
                                         is_absolute={isMobile()}
                                         label={localize('Next')}
-                                        onCancel={() => handleCancel(values)}
                                     />
```

Comment:
`cancel_label` and `onCancel()` are useless in this case because the `FormSubmitButton` component utilises values from these props only when `has_cancel` is also passed to it. I've also made all related changes to the `CFDPersonalDetailsForm` props respectively.

Reply:
I suggest reverting back this changes and adding them to a separate card, this card is meant to add test cases for a component and not to refactor another component.

Reply:
There are 2 reasons why I’m doing it together with the test:

- In the test, I have to specify the components props including those which are not used and have to be removed, so after the PR for refactoring the component is merged (basically I can create it now very fast as a separate PR), we’ll have to modify the test again.
- Until refactoring PR is merged, the test is going to have less coverage because the unused code cannot be actually tested since it’s unreachable

Is it ok? If yes, I’ll proceed with your suggestion 🙂